tests/model_serving/model_server/private_endpoint/constants.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List, Any
+
+AWS_REGION: str = "us-east-2"
+AWS_BUCKET: str = "ods-ci-wisdom"
```

Comment:
please see https://github.com/opendatahub-io/opendatahub-tests/pull/21/files#diff-8df5ee07ff5e7eb9edcc5490df50302e439eb5f8e62fe669f159ce71b3407880R13-R15

this type of config should go there

Reply:
I'm not sure I agree on this point, what if I want to define different variables for different test contexts without having to pass them in or define them as environment variables all the time? I think having some of these set as constants in the code works fine